ansible_collections/arista/avd/plugins/action/anta_catalog_builder.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+from typing import Any, Dict, List, Tuple
+
+from ansible.plugins.action import ActionBase
+from anta.loader import parse_catalog
+
+TestTuple = Tuple[str, Dict[str, str]]
+Catalog = Dict[str, List[Dict[str, Any]]]
+
+
+AVD_MAPPING: Dict[str, List[TestTuple]] = {
+    "management_api_http": [("anta.tests.security.VerifyAPIHttpStatus", {}), ("anta.tests.security.VerifyAPIHttpsSSL", {"profile": "https_ssl_profile"})]
+}
+
+
+def _get_package_name(test_name: str) -> str:
```

Comment:
please move all your functions and constants into a lib you can import from plugin_utils. That will help us reuse the code in `pyavd`.

Reply:
10/4